src/components/Table/TableConfirmButtonsRow.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { createPortal } from 'react-dom';
+import { ConfirmButton, CancelButton } from '../InlineEdit';
+import { noop } from '../../common/helpers';
+
+class TableConfirmButtonsRow extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+
+    this.saveRowDimensions = this.saveRowDimensions.bind(this);
+    this.handleScroll = this.handleScroll.bind(this);
+    this.handleResize = this.handleResize.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchClientDimensions();
+    window.addEventListener('scroll', this.handleScroll);
+    window.addEventListener('resize', this.handleResize);
+  }
```

Comment:
These should use `helpers.debounce`

Reply:
Fixed, both functions are now debounced in the constructor.